eng/source-build-patches/0001-WIP-for-source-build.patch

```diff
@@ -0,0 +1,491 @@
+From 48442474744a416eee4f367d3e389a0916047842 Mon Sep 17 00:00:00 2001
+From: Omair Majid <omajid@redhat.com>
+Date: Thu, 29 Oct 2020 18:20:09 -0400
+Subject: [PATCH 1/4] WIP for source-build
+
+This is a grab-bag of fixes to make roslyn-analzyers build in a
+source-build context.
+
+- Fix up TargetFrameworks:
+  - Remove net46, net47, etc
+  - Move from netstandard 1.3 to 2.0
+
+- Remove FxCopAnalyzers.sln, which is not needed for source-build
+
+---
+ RoslynAnalyzers.sln                           | 256 ---------------
+ eng/Versions.props                            |   4 +
+ eng/common/dotnet-install.sh                  |   0
+ 37 files changed, 436 insertions(+), 492 deletions(-)
+ delete mode 100644 FxCopAnalyzers.sln
+ mode change 100644 => 100755 eng/common/dotnet-install.sh
+
+diff --git a/RoslynAnalyzers.sln b/RoslynAnalyzers.sln
+index 82e87f32e..6807ca029 100644
+--- a/RoslynAnalyzers.sln
++++ b/RoslynAnalyzers.sln
```

Comment:
❗ We should not patch **RoslynAnalyzers.sln**. If you need to build a subset of projects, create a **SourceBuild.slnf** solution filter, which is derived from **RoslynAnalyzers.sln** and only contains the projects needed for the build.